config.md

```diff
@@ -101,6 +101,17 @@ For POSIX platforms the `mounts` structure has the following fields:
 * **`type`** (string, OPTIONAL) The type of the filesystem to be mounted.
     * Linux: filesystem types supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660"). For bind mounts (when `options` include either `bind` or `rbind`), the type is a dummy, often "none" (not listed in */proc/filesystems*).
     * Solaris: corresponds to "type" of the fs resource in [zonecfg(1M)][zonecfg.1m].
+* **`uidmappings` (array of type LinuxIDMapping, OPTIONAL) The file system mapping to convert uid from value written in the container's image into desired value.
+This mapping contains of following fields: ContainerID, HostID, Size.
+These values would be written into temporary user namespace's /proc/$pid/uid_map as followes "ContainerID HostID Size".
+The uid of the image file system would be mapped inside kernel from ContainerdID to HostID.
+The Size field is a range for mapping, as an example.
+        1000 1001 1       -> map uid 1000 to uid 1001 in <target> and no other ids
+        20000 100000 1000 -> map uid 20000 to uid 100000, uid 20001 to uid 100001 [...] until uid 20999 in <target>
+Currently up to 340 separate idmappings may be specified by the kernel.
+* **`gidmappings` (array of type LinuxIDMapping, OPTIONAL) The file system mapping to convert gid from value written in the container's image into desired value.
+For more details see `uidmappings`.
+
```

Comment:
please use **uidMappings** and **gidMappings** (with uppercase M and the missing closing **).

We shouldn't mention "container's image" since the mapping is performed from the source file system (and runtime specs don't know about images).

I'd use something simpler as the following:

```suggestion
* **`uidMappings`** (array of type LinuxIDMapping, OPTIONAL) The mapping to convert UIDs from the source file system to the destination mount point.
This mapping contains the following fields: ContainerID, HostID, Size.
The Size field is a range for mapping, as an example.
        1000 1001 1       -> map UID 1000 to uid 1001 in <target> and no other UIDs
        20000 100000 1000 -> map UID 20000 to uid 100000, uid 20001 to UID 100001 [...] until UID 20999 in <target>
* **`gidMappings`** (array of type LinuxIDMapping, OPTIONAL) The mapping to convert GIDs from the source file system to the destination mount point.
For more details see `uidMappings`.

```